client/src/components/pages/MembersPage/components/MembersVisualisation.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styled from 'styled-components'
+import Member from './MemberCard'
+
+export function Visualisation({ members, deleteMember }) {
+  return (
+    <VisualisationStyled>
+      {members &&
+        members.map(member => (
+          <Member
+            key={member.id + 'visualisation'}
+            member={member}
+            deleteMember={deleteMember}
+          />
+        ))}
+    </VisualisationStyled>
+  )
+}
+
+export function VisualisationDisabled({ members }) {
```

Comment:
Maybe refactor to one component that gets passed a disabled prop, instead of two different components?